gammapy/estimators/utils.py

```diff
@@ -686,3 +688,68 @@ def get_rebinned_axis(fluxpoint, axis_name="energy", method=None, **kwargs):
         edges = np.append(edges_min, edges_max[-1])
         axis_new = MapAxis.from_edges(edges, name=axis_name, interp=ax.interp)
     return axis_new
+
+
+def get_combined_significance_maps(estimator, datasets):
```

Comment:
Is there a specific reason for such an interface?
A priori you could simply combine the results of estimators run independently no? 
Is it to ensure that significance maps where obtained in a consistence manner?

Reply:
yes consistency was the goal.
The function could accept both and use directly a list of flux maps if provided.
Alternatively I could use only flux maps here and modify the estimator.run to return a list of results if a datasets is given (and eventually parallelise that loop in a separate PR).